docker/appserver/WebSphere/src/scripts/configureJdbc.py

```diff
@@ -71,7 +68,16 @@ def createDatasource(datasourceName, providerName, authAlias, properties):
 	])
 
 
+createDatasource('ibis4test-mysql', 'MySQL JDBC Driver', authAliasName, [
+		[['name', 'URL'],  ['value', 'jdbc:mysql://host.docker.internal:3307/testiaf']],
+		[['name', 'sslMode'], ['value', 'DISABLED']]
+	])
+
+createDatasource('ibis4test-mariadb', 'MariaDB JDBC Driver', authAliasName, [
+		[['name', 'URL'],  ['value', 'jdbc:mysql://host.docker.internal:3306/testiaf']],
+		[['name', 'sslMode'], ['value', 'DISABLED']]
+	])
```

Comment:
in de tomcat context staat ook nog als properties:
```
		serverTimezone="Europe/Amsterdam"
		allowPublicKeyRetrieval="true"
		pinGlobalTxToPhysicalConnection="true"
		socketTimeout="5000"
```
Hoeven deze properties op WebSphere niet gezet te worden?

Reply:
De testen mislukten niet als ze niet gezet waren, maar ik heb ze toegevoegd.